<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">* {
  border: none;
  background-color: transparent;
  color: #FFF; /* Specify a default color */
}

#centralwidget {
  background-color: white;

}

#side_menu {
  background-color: #071e26;
  border-radius: 20px;
}

QPushButton {
  padding: 10px;
  background-color: #071e26;
  border-radius: 5px;
}

#main_body {
  background-color: #071e26;
  border-radius: 10px;
}

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="header">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Menu</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/icons_folder/align-justify.svg</normaloff>:/icons/icons_folder/align-justify.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Arial Black</family>
                <pointsize>15</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: black;</string>
              </property>
              <property name="text">
               <string>My app</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QWidget" name="side_menu" native="true">
           <property name="maximumSize">
            <size>
             <width>1677212</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Item 1</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/icons/icons_folder/airplay.svg</normaloff>:/icons/icons_folder/airplay.svg</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>Item 2</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icons/New folder/award.svg</normaloff>:/icons/New folder/award.svg</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>Item 3</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icons/New folder/coffee.svg</normaloff>:/icons/New folder/coffee.svg</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_5">
                 <property name="text">
                  <string>Item 4</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icons/New folder/music.svg</normaloff>:/icons/New folder/music.svg</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_6">
                 <property name="text">
                  <string>Item 5</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/icons/New folder/wifi.svg</normaloff>:/icons/New folder/wifi.svg</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="main_body">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">font: 8pt &quot;Nirmala UI&quot;;
font-weight:500;</string>
                </property>
                <property name="text">
                 <string>Upload Notes:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="select_button">
                <property name="styleSheet">
                 <string notr="true">font: 8pt &quot;Nirmala UI&quot;;
font-weight:500;
border-bottom:1px solid white;
border-radius:none;</string>
                </property>
                <property name="text">
                 <string>Select Files</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item alignment="Qt::AlignLeft">
                 <widget class="QLabel" name="label_5">
                  <property name="styleSheet">
                   <string notr="true">font: 8pt &quot;Nirmala UI&quot;;
font-weight:500;</string>
                  </property>
                  <property name="text">
                   <string>Page:</string>
                  </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignLeft">
                 <widget class="QLineEdit" name="page_form">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>38</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);
color:black;
border-radius:2px;
font: 8pt &quot;Nirmala UI&quot;;
font-weight:500;</string>
                  </property>
                  <property name="placeholderText">
                   <string>Page No</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="styleSheet">
                   <string notr="true">font: 8pt &quot;Nirmala UI&quot;;
font-weight:500;</string>
                  </property>
                  <property name="text">
                   <string>Title</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="title_form">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>38</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);
color:black;
font: 8pt &quot;Nirmala UI&quot;;
font-weight:400;
border-radius:5px;
font-weight:500;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="styleSheet">
                 <string notr="true">font: 8pt &quot;Nirmala UI&quot;;
font-weight:500;</string>
                </property>
                <property name="text">
                 <string>Description:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="Description_form">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color:black;
border-radius:2px;
font: 8pt &quot;Nirmala UI&quot;;
font-size:20px;
font-weight:500;</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">font: 8pt &quot;Nirmala UI&quot;;
font-size:20px;
font-weight:700;</string>
              </property>
              <property name="text">
               <string>Upload Chapter Notes</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="upload_button">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(3, 146, 255);
font: 8pt &quot;Nirmala UI&quot;;
font-weight:600;</string>
              </property>
              <property name="text">
               <string>Upload</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">font: 8pt &quot;Nirmala UI&quot;;
font-weight:600;</string>
                </property>
                <property name="text">
                 <string>Chapter Number</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Chapter_form">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>38</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 8pt &quot;Nirmala UI&quot;;
border-radius:5px;
font-weight:500;</string>
                </property>
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
        <zorder>main_body</zorder>
        <zorder>side_menu</zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
